static/app/styles/overflowEllipsis.tsx

```diff
@@ -1,5 +1,8 @@
 /**
- * CSS styles used to add a ellipsis overflow
+ * CSS styles used to add a ellipsis overflow. This is deprecated, use the
```

Comment:
since this is no longer being used, should we remove this file?

Reply:
hmm I plan to remove this file within a few weeks, once everyone learns of this deprecation and the new location for `overflowEllipsis` (the theme object). Also, we still use this file in `getsentry`, so I thought it would be better to just deprecate it, remove the imports in `getsentry`, then remove the file.